### ISSUE TITLE
StatsFuns Compat?

### ISSUE THREAD

**Issue by oxinabox** [malmaud/TensorFlow.jl#298]
Should we overload some things from the StatsFuns namespace?

Right now, we overload things basically only from Base, when it comes to operations.

There are a couple fairly common neural netfunctions that are in StatsFuns.

namely

 - logitstic == sigmoid
 - softmax
 - softplus (I think)

Overloading these names with eg `logistic(z::AbstractTensor) = nn.sigmoid(z)`
would make for more transparent operations.

I StatsFuns is already in our REQUIRE tree, so it is no true new dependency.

**Comment by malmaud** [comment]
Ya, good idea. I support this.
